build.sh

```diff
@@ -22,6 +23,10 @@ BUILD_GENERATOR_DEFAULT="Unix Makefiles"
 BUILD_GENERATOR="$BUILD_GENERATOR_DEFAULT"
 MAKE_TASKS=-1
 ADDRESS_SANITIZER=0
+DO_CLEAN_BUILD=0
+DO_CLEAN_INSTALL=0
+MANIFEST_FILE="$BUILD_DIR/install_manifest.txt"
```

Comment:
I don't see any update for this file. Always empty? How can this be used to clean install directory?

Reply:
This file is generated by cmake when the build is done. It's not under version control.

Reply:
I don't have this file after a successful build using build.sh.

Reply:
Even in the build/ directory when you launch with --install ?